src/sortStudents.ts

```diff
@@ -1,16 +1,52 @@
 
 export interface Student {
-  // describe Student interface
+  name: string;
+  surname: string;
+  age: number;
+  married: boolean;
+  grades: number[];
 }
 
 export enum SortType {
-  // describe SortType enum
+  Name = 'name',
+  Surname = 'surname',
+  Age = 'age',
+  Married = 'married',
+  AverageGrade = 'grades',
 }
 
-// create SortOrder type
-export type SortOrder;
+export enum SortOrder {
+  asc = 'asc',
+  desc = 'desc',
+}
```

Comment:
```suggestion
export enum SortOrder {
  Asc = 'asc',
  Desc = 'desc',
}
```
use capitalized naming for enums